jupyterhub/values.yaml

```diff
@@ -82,6 +82,8 @@ proxy:
     type: letsencrypt
     letsencrypt:
       contactEmail: ''
+    key:
```

Comment:
this should be manual.key and manual.cert IMO. I think we should be doing:

```
type: X
X:
   <all-X-related-options>
```

Reply:
done